nsot/middleware/request_logging.py

```diff
@@ -16,12 +16,16 @@ def process_request(self, request):
         return None
 
     def process_response(self, request, response):
+        if 'HTTP_X_FORWARDED_FOR' in request.META: #If
+            request_ip_path = '%s,%s' % (request.META.get('REMOTE_ADDR'), request.META.get('HTTP_X_FORWARDED_FOR'))
```

Comment:
I see you've chosen to make this a 2-tuple of (original, forwarded). Is this common practice? I feel like I'd rather just see the one IP.